client/reader/list-manage/export-button.jsx

```diff
@@ -0,0 +1,66 @@
+/**
```

Comment:
Good call 👍 

I'd refactored the `<button>` so it could be used between the two, but they share more functionality than that (like the blob creation/download process). 

I've reworked the block to support both types of export.